libs/blocks/instagram/instagram.js

```diff
@@ -0,0 +1,19 @@
+import { createIntersectionObserver, createTag } from '../../utils/utils.js';
+
+export default function init(a) {
+  const embedInstagram = () => {
+    if (!a.origin?.includes('instagram')) return;
+
+    const anchor = createTag('a', { href: a.href });
+    const blockquote = createTag('blockquote', { class: 'instagram-media' }, anchor);
+    const wrapper = createTag('div', { class: 'embed-instagram' }, blockquote);
+    const head = document.querySelector('head');
```

Comment:
If you use createScript out of utils, it will automatically de-dupe the script for you.